www/js/diary/details/TripSectionsDescriptives.tsx

```diff
@@ -2,7 +2,6 @@ import React, { useContext } from 'react';
 import { View, StyleSheet } from 'react-native';
 import { Icon, Text, useTheme } from 'react-native-paper';
 import useDerivedProperties from '../useDerivedProperties';
-import { getBaseModeByValue } from '../diaryHelper';
```

Comment:
This function still exists in the diaryHelper, right? Are you removing the usages now so that you can remove it from the helper later?

Reply:
Yes, that's what I was thinking – but I now see that there were no usages left so I have removed it altogether